server/resource/allUsers.py

```diff
@@ -0,0 +1,15 @@
+from flask_restful import Resource
+from flask_jwt_extended import (jwt_required)
+import sys
+sys.path.append("..")
```

Comment:
Is this sys path modifications necessary? If so, there may be something wrong with the way you are doing imports in Python